### PR TITLE
ERC20 credit debit tracking

### PR DESCRIPTION
**Pull request by d4v1d03**
The only issue right now is with the unique_id, at the end of the unique id we have to concate credit/debit depending on the transaction type? How to we decide that because value will always remain positive for and to-from address which we have kept.

### CODE REVIEW

File: CreditDebit/studio.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - Event: EIP712DomainChanged()
+        Handler: src/handlers/blockflow/EIP712DomainChanged.EIP712DomainChangedHandler
+      - Event: MinterUpdated(address indexed,address indexed)
+        Handler: src/handlers/blockflow/MinterUpdated.MinterUpdatedHandler
+      - Event: OwnershipTransferStarted(address indexed,address indexed)
+        Handler: >-
+          src/handlers/blockflow/OwnershipTransferStarted.OwnershipTransferStartedHandler
+      - Event: OwnershipTransferred(address indexed,address indexed)
+        Handler: >-
+          src/handlers/blockflow/OwnershipTransferred.OwnershipTransferredHandler
```
Comment (pradyuman-verma) [inline CreditDebit/studio.yaml:28]: can remove these handlers

File: CreditDebit/src/handlers/blockflow/Transfer.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    rawAmount: value,
+    rawAmountString: value.toString(),
+    amount: amount,
+    amountString: amount.toString(),
+    transactionHash: transaction.transaction_hash,
+    logIndex: log.log_index,
+    blockTimestamp: block.block_timestamp,
+    blockNumber: block.block_number,
+    blockHash: block.block_hash,
+  });
```
[the remark this answers is not in the document]
Comment (d4v1d03) [reply CreditDebit/src/handlers/blockflow/Transfer.ts:53]: okay ill create another table for that , should the amount be in negative for that or should i interchange the address and counterpartyaddress for them ?

File: CreditDebit/studio.yaml (hunk 1 above)
Comment (d4v1d03) [reply CreditDebit/studio.yaml:28]: sorry, missed that issue

File: CreditDebit/src/handlers/blockflow/Transfer.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (pradyuman-verma) [reply CreditDebit/src/handlers/blockflow/Transfer.ts:53]: Yes, in the same db.

**Comment by d4v1d03** [comment]
working tests 
wokring_credit_debit

**Comment by pradyuman-verma** [comment]
I think we can merge it inside ERC20 handlers?

**Review by pradyuman-verma — APPROVED** [review]

**Pull request merged**